syntax = "proto3";

package syft.lib.torch;

message ParameterProto {
  bytes tensor = 1;
  bool requires_grad = 2;
  optional bytes grad = 3;
  optional bytes grad_sample = 4; // opacus
}
